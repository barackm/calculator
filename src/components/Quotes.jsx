import React from 'react';

const Quotes = () => (
  <div className="quotes-main-container">
    <div className="overlay-calculator quotes" />
    <div className="quote-area d-flex justify-content-center align-items-center flex-column">
      <h3 className="quote-title">Mathmatical Quotes</h3>
      <hr className="diviser" />
      <p className="text-center">
        Mathematics is not about numbers, equations, computations, or
        algorithms: it is about understanding
        <br />
        <i>William Paul Thurston</i>
      </p>
    </div>
  </div>
);

export default Quotes;
