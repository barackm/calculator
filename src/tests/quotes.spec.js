import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Quotes';

describe('Quotes Component', () => {
  it('Should match spanshot', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
