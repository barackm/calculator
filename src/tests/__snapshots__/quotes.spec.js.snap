// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes Component Should match spanshot 1`] = `
<div
  className="quotes-main-container"
>
  <div
    className="overlay-calculator quotes"
  />
  <div
    className="quote-area d-flex justify-content-center align-items-center flex-column"
  >
    <h3
      className="quote-title"
    >
      Mathmatical Quotes
    </h3>
    <hr
      className="diviser"
    />
    <p
      className="text-center"
    >
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding
      <br />
      <i>
        William Paul Thurston
      </i>
    </p>
  </div>
</div>
`;
